.repo-metadata.json

```diff
@@ -6,6 +6,7 @@
   "issue_tracker": "https://issuetracker.google.com/savedsearches/559663",
   "release_level": "beta",
   "language": "python",
+  "library_type": "AGENT",
```

Comment:
I think this is a `GAPIC_MANUAL` https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest

Reply:
Fixed.